js/gracelib.js

```diff
@@ -3098,7 +3098,7 @@ function callmethodChecked(obj, methname, argcv) {
         } else if (e.exctype === 'graceexception') {
             e.exitStack.unshift({
                 className: obj.className,
-                methname: methname,
+                methname: canonicalMethodName(methname),
```

Comment:
The above suggestion also means that the conversion need happen in just one place, not two.